.github/workflows/prepare-for-minor-release.yml

```diff
@@ -121,10 +118,6 @@ jobs:
          php-version: '8.1'
          tools: composer:v2
 
-    - name: Update dev branch alias via Composer
-      run: |
-        composer config extra.branch-alias.dev-main ${{ inputs.release_branch_name }}-dev
-
```

Comment:
Verified that this step is no longer needed (you can see that no changes to composer.json show up in the `git diff` step of the last workflow run)